src/table/components/head/TableHeadWrapper.tsx

```diff
@@ -31,10 +33,18 @@ function TableHeadWrapper({ tableData, areBasicFeaturesEnabled }: TableHeadWrapp
           // when nebula does not handle keyboard navigation
           const isActive = layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === column.colIdx;
           const ariaSort = isActive ? FullSortDirection[column.sortDirection] : undefined;
+          const isLastColumn = columnIndex === columns.length - 1;
+
+          const widthStyle = {
+            ...(areBasicFeaturesEnabled && {
+              width: (columnWidths[columnIndex] || 200) - (isLastColumn ? 8 : 9), // subtract padding and border
```

Comment:
What is 200? Is that the default value?

Reply:
Consider making all numerics here constants so that they can be named after what they are (no need to have comment) and re-used elsewhere